src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py

```diff
@@ -0,0 +1,403 @@
+        return self.get_input_debug_name(index)
+
+    def get_input_shape(self, index):
+        if index < len(self.input_shapes):
+            return PartialShape(self.input_shapes[index])
+        input = self._raw_input(index)
+        return self.get_shape_for_value(input)
+
+    def get_input_type(self, index):
+        if index < len(self.input_types):
+            return OVAny(pt_to_ov_type_map[str(self.input_types[index])])
+        input = self._raw_input(index)
+        return self.get_type_for_value(input)
+
+    def get_output_debug_name(self, index):
+        return "output"+str(index)
+
+    def get_output_shape(self, index):
+        output = self._raw_output(index)
+        return self.get_shape_for_value(output)
+
+    def get_output_type(self, index):
+        output = self._raw_output(index)
+        return self.get_type_for_value(output)
+
+    def _get_known_type_for_value(self, type):
+        '''
+            Returns known/unknown types wrapped as OVAny
+        '''
+        # Check for simple scalar types first
+        # TODO: Don't use str, use native types
+        if type is None:
+            return OVAny(OVType.dynamic)
+        if str(type) in pt_to_ov_type_map:
+            return OVAny(pt_to_ov_type_map[str(type)])
+        elif type.__class__ is torch.TensorType:
+            # Tensor type, parse element type
+            # TODO: replace string by native type
+            # return OVAny(PartialShape([1,2,3]))
+            return OVAny(DecoderType.Tensor(self._get_known_type_for_value(type.dtype())))
+        elif type.__class__ is torch.ListType:
+            element_type = type.getElementType()
+            return OVAny(DecoderType.List(self._get_known_type_for_value(element_type)))
+        else:
+            # Not yet recognized
+            return OVAny(OVType.dynamic)
+            #pt_type_class = value.type().__class__
+            #    if pt_type_class is torch.ListType:
+
+    def get_shape_for_value(self, value):
+        if value and ('tensor_meta' in value.meta.keys()):
+            return PartialShape(value.meta['tensor_meta'].shape)
+        return PartialShape([1])
+
+    def get_type_for_value(self, value):
+        if issubclass(type(value), torch.fx.Node):
+            if ('tensor_meta' in value.meta.keys()):
+                pt_type = value.meta['tensor_meta'].dtype
+                if str(pt_type) in pt_to_ov_type_map:
+                    ov_type = pt_to_ov_type_map[str(pt_type)]
+                    return OVAny(ov_type)
+            else:
+                return OVAny(OVType.f32)
+        elif isinstance(value, int):
+            return OVAny(OVType.i32)
+        elif isinstance(value, float):
+            return OVAny(OVType.f32)
+        elif isinstance(value, bool):
+            return OVAny(OVType.boolean)
+        else:
+            return OVAny(OVType.f32)
+
+    def get_input_transpose_order(self, index):
+        return []
```

Comment:
is there any plans to make this and the next methods work?

Reply:
Yeah after the pre-release we plan to look into this